README.md

```diff
@@ -85,6 +85,9 @@ select * from db1.table1;
   build the image locally.
 - `docker run -p 8181:8181 -p 8182:8182 apache/polaris:latest` - To run the image.
 
+The Polaris codebase contains some docker compose examples to quickly get started with Polaris,
+using different configurations. Check the `./docker` directory for more information.
```

Comment:
There are Spark and Trino docker compose examples under the directory `./getting-started`. Can we consolidate all docker compose examples into one place?

Reply:
Of course, great idea! I'm sorry I didn't think about it before 🤦‍♂️ 

I will move everything to `.getting-started`.

Reply:
Yes good point. I think it makes sense to gather everything in getting started and optionally point to getting started.